tests/clipboard/clipboard_plugin.test.ts

```diff
@@ -2001,6 +2005,127 @@ describe("clipboard", () => {
     expect(getCell(model, "C3")!.content).toBe(expectedInvalidFormula);
   });
 
+  test("filtered rows are ignored when copying range", () => {
+    const model = createModelFromGrid({
```

Comment:
nitpick: would be prettier/shorter with a  prettier-ignore and a grid-like display